src/ips/graph_utils.rs

```diff
@@ -79,7 +80,23 @@ pub fn construct_graph(nodes: &[Node]) -> Graph<SocketAddr> {
 
     for node in nodes {
         let node_addr = node.addr;
+
+        // This is a hack to add nodes that are not connected to any other node.
```

Comment:
Let's mention this can happen when certain nodes are found through a different network (e.g. not the most responsive zcash nodes discovered via the flux nodes)

Reply:
Mentioned.